extensions/objects/templates/set-of-normalized-string-editor.component.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for list of normalized string editor.
+ */
+
+// Every editor directive should implement an alwaysEditable option. There
```

Comment:
This doesn't seem to be implemented by ``list-editor.directive.html``. Given the comment, should it be?

Reply:
I'm not sure- I don't think it applies anymore given that most of the editor directives don't.